app/services/schemas.py

```diff
@@ -1,7 +1,7 @@
 from pydantic import BaseModel
 from typing import Optional, List, Any
 from enum import Enum
-from app.services.tool_registry import BaseTool
+from services.tool_registry import BaseTool
```

Comment:
Fetch these utility functions from the `app` folder.

Reply:
fixed it